python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py

```diff
@@ -0,0 +1,44 @@
+"""
+Reads a file, breaks each line into words, strips whitespace and
+punctuation from the words, and converts them to lowercase.
+"""
+
+
+import os
+from helpers.file_reader import read_file
+from constants.constants import TEST
+
+
+def clean_and_split_text(test):
```

Comment:
Please update the function and parameter name.

Reply:
Done